gitpod-io/gitpod
# [server] Use content-service

This PR removes the dependency to the remote storage configuration from the server component. Instead, it uses the content-server component.

Fixes https://github.com/gitpod-io/gitpod/issues/3144

### How to test

The following functions are affected and should be tested with MinIO and GCP storage:
- Installing a VSCode extension (for the workspace not the user to have it for the next step)
- Start a workspace with a previous installed VSCode extension (just stop and restart the workspace from prev. step)
- Download workspace data from the dashboard
- Delete a workspace (after pressing the delete button change the soft deletion time in the DB, e.g `update d_b_workspace set softDeletedTime = '2020-01-01 01:30:30';`, here for all workspaces; wait for max. 30 min. to let the GC run (see server log), verify in MinIO or GCP storage that the workspace backup is gone)
- Delete a user account (verify in MinIO or GCP storage that the user bucket is gone), should be tested with a non-empty user bucket!

***

- Minio can be accessed at https://minio.corneliusludmann-server-move-all-3144.staging.gitpod-dev.com/ 
- GCP storage can be tested with the werft flag `/werft storage=gcp`
